packages/suite/src/support/messages.ts

```diff
@@ -9018,6 +9018,10 @@ export default defineMessages({
         id: 'TR_MOBILE_APP_PROMO_TEXT_FOOTER',
         defaultMessage: 'Sync & track on your phone with <b>Trezor Suite Lite</b>',
     },
+    TR_DASHBOARD_REFERRAL_BUTTON: {
+        id: 'TR_DASHBOARD_REFERRAL_BUTTON',
+        defaultMessage: 'Earn $20 per Referral',
```

Comment:
A weird Q: Shouldnt this amount here from referal be somehow tied to the link itself? Like, what if we change the it? Shouldn't it be obivous somehow from the code that this text + amount belong to this particular referal link?

Reply:
Maybe you can use message system for it (feature flag)